coalib/bears/Bear.py

```diff
@@ -274,6 +295,32 @@ def log_message(self, log_message, timestamp=None, **kwargs):
     def run(self, *args, dependency_results=None, **kwargs):
         raise NotImplementedError
 
+    def debugger_config(self,*args,**kwargs):
+        debug_bears = kwargs.get('debug_bears')
+        kwargs.pop('debug_bears')
+        if ('linter' in str(self)) and \
```

Comment:
I would rather rewrite `debugger_config` in `@linter` so it can implement its own debugger configuration. The `Bear` base class shouldn't know about `linter`.

Reply:
I have to check where in `Linter` file I can call it :+1:

Reply:
@Makman2  I think I have made one mistake here, check that case `1` and case `3` are calling same functions here, so 

IMO without case 1, it will do the same thing, so now Bear Base class will not know about linter :+1:

Reply:
So writing the `debugger_config` in `@linter` will not be needed now.